### PR TITLE
Fontawesome updates

### PR DESCRIPTION
**Pull request by benkeen**
See individual commit messages.

### CODE REVIEW

File: assets/less/icons.less [hunk 1]
```diff
@@ -18,21 +18,23 @@
...
-  src: url("../fonts/fauxtonicon.eot?#iefix") format("embedded-opentype"),
-       url("../fonts/fauxtonicon.woff") format("woff"),
-       url("../fonts/fauxtonicon.ttf") format("truetype"),
-       url("../fonts/fauxtonicon.svg#fauxtonicon") format("svg");
+  font-family: "fauxtonicon1";
+  src: url("../fonts/fauxtonicon1.eot");
+  src: url("../fonts/fauxtonicon1.eot?#iefix") format("embedded-opentype"),
+       url("../fonts/fauxtonicon1.woff") format("woff"),
+       url("../fonts/fauxtonicon1.ttf") format("truetype"),
+       url("../fonts/fauxtonicon1.svg#fauxtonicon1") format("svg");
```
Comment (robertkowalski) [inline assets/less/icons.less]: what do you think of putting the font-name into a variable, so it just has to get changed once?

Comment (benkeen) [inline assets/less/icons.less]: Actually it is! It's stored in the yaml file in the same folder. `font_name` or something: this is the generated less content.

**Comment by robertkowalski** [comment]
ah cool

+1

**Comment by benkeen** [comment]
Merged as 62f149a

**Pull request merged**